.genx/package-lock.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "{{groupId}}.pbc-{{appName}}-genx",
```

Comment:
I think we need one here as well, because here we want something like '@genesislcap', whereas I think this group id is more in the java standard way of `com.mycompany` ?